src/Dapr.Client/DaprClient.cs

```diff
@@ -783,6 +783,75 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             StateOptions stateOptions = default,
             IReadOnlyDictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
+        /// <summary>
+        /// Define a SaveStateAsync task that gets a binary input   
+        /// </summary>
+        /// <param name="storeName">The  </param>
+        /// <param name="key"></param>
+        /// <param name="binaryValue"></param>
+        /// <param name="stateOptions"></param>
+        /// <param name="metadata"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public abstract Task SaveStateByteAsync(
+                       string storeName,
+                       string key,
+                       byte[] binaryValue,
```

Comment:
In the past, we've used `ReadOnlyMemory<byte>` instead of just a raw `byte[]`. Let's change it here to be consistent.

Reply:
I have changed SaveStateByteAsync, TrySaveStateByteAsync to use ReadOnlyMemory<byte>. Thanks for your review.